docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,104 @@
+Using a virtual computer allows you to develop and run analyses that require more CPUs and memory than is available on your local machine.
+
+_Note that you will need access to AWS to use LSfR._
+See [getting access to AWS](../../getting-started/accessing-resources/index.md#getting-access-to-aws) to learn how to get set up with AWS.
+
+- The virtual computers provided through LSfR use an Ubuntu operating system.
+- During setup, you will choose your desired configuration, including the amount of virtual CPUs, memory, and storage space.
+    - You also have the option to add [additional storage to your virtual computer](STUB-LINK for volumes).
+- All virtual computers will have a set of pre-installed applications needed for working with OpenScPCA, including:
+    - Git and GitKraken
+    - The AWS command line interface
+    - R
+    - Conda
+
+!!! note "More on virtual computers"
+
+    Read more about [virtual computers with Lightsail for Research](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/computers.html).
+
+## How to create a virtual computer
+
+Follow the below steps to create a virtual computer to use with LSfR.
+
+1. Navigate to the [access portal URL from when you set up your user in IAM Identity center](../aws/index.md#joining-iam-identity-center).
+This link should bring you to the AWS Console and home page.
+
+    Once here, be sure that you are in the `us-east-2` region, by selecting the drop-down menu in the tool bar.
+
+    <figure markdown="span">
+        ![Select region](../../img/creating-vcs-1.png){width="400"}
+    </figure>
+
+1. You will need to use the AWS Service Catalog to create virtual computers.
+Open the AWS Service Catalog by using the search bar and typing, `service catalog`.
+
+    <figure markdown="span">
+        ![Select service catalog](../../img/creating-vcs-2.png){width="600"}
+    </figure>
+
+1. Select `LightsailInstance` in the product list, and click `Launch product`.
+
+    <figure markdown="span">
+        ![Launch product](../../img/creating-vcs-3.png){width="600"}
+    </figure>
+
+1. You will then choose the name and configurations for your virtual computer.
+
+    - Start by providing a `Provisioned product name`.
+    <!--TODO Do we want to provide guidance on names?-->
+    - Pick an `Application` to use from the drop-down menu.
+    This installs any additional applications along with the pre-installed applications.
+        - If you are planning to develop in R, we recommend choosing `Rstudio`, which includes RStudio.
+        - If you are planning to develop in Python, we recommend choosing `VSCodium`, which includes VSCodium, a distribution of Microsoft's editor VS Code.
+        - For information on all application options, see [the LSfR documentation on Applications](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/blueprints-plans.html).
+    - Choose any of the availability zones.
```

Comment:
I think we might need to recommend `us-east-2a` because if your volume is not in the same AZ (which is not at all transparent to you as a user!), you can't attach it, which does make sense to me.

We should probably remove the other options eventually.